data/json/mapgen/cosmics_additions/LANCenter.json

```diff
@@ -0,0 +1,168 @@
+        "_": "t_grass",
+        "a": "t_console_broken",
+        "c": "t_floor",
+        "d": "t_dirt",
+        "s": "t_sidewalk",
+        "v": "t_floor",
+        "w": "t_window",
+        "{": "t_floor",
+        "|": "t_wall"
+      }
+    },
+    "om_terrain": "lancenter",
+    "type": "mapgen",
+    "weight": 100
+  },
+  {
+    "method": "json",
+    "object": {
+      "furniture": {
+        "#": "f_table",
+        "&": "f_trashcan",
+        "+": "f_null",
+        "-": "f_null",
+        ".": "f_null",
+        "6": "f_null",
+        "C": "f_counter",
+        "D": "f_null",
+        "H": "f_armchair",
+        "P": "f_sign",
+        "S": "f_sink",
+        "_": "f_null",
+        "c": "f_chair",
+        "d": "f_null",
+        "f": "f_fridge",
+        "p": "f_null",
+        "s": "f_null",
+        "t": "f_toilet",
+        "v": "f_vending_c",
+        "{": "f_glass_fridge",
+        "|": "f_null"
+      },
+      "place_items": [
+        { "chance": 45, "item": "fridge", "x": 20, "y": 20 },
+        { "chance": 50, "item": "vending_food_items", "x": 14, "y": 20 },
+        { "chance": 15, "repeat": [ 1, 3 ], "item": "trash", "x": 19, "y": 15 },
+        { "chance": 50, "item": "vending_drink_items", "x": 15, "y": 20 },
+        { "chance": 50, "item": "vending_drink_items", "x": 16, "y": 20 }
+      ],
+      "place_toilets": [ { "x": 3, "y": 20 } ],
+      "rows": [
+        "ssssssssssssssssssssssss",
+        "ssssssssssssssssssssssss",
+        "__||---|||---||ppppppp_d",
+        "__|...........|ppppppp__",
+        "__|..c.c.c.c..-ppppppp__",
+        "_d|.#6#6#6#6#.-ppppppp__",
+        "__|.#6#6#6#6#.-ppppppp__",
+        "__|..c.c.c.c..|ppppppp__",
+        "__|...........|ppppppp__",
+        "__|..c.c.c.c..|pppppppd_",
+        "__|.#6#6#6#6#.-ppppppp__",
+        "__|.#6#6#6#6#.-ppppppp__",
+        "__|..c.c.c.c..-ppppppp__",
+        "__|...........|ppppppp__",
+        "__|..c.c.c.c..||+|P-||__",
+        "__|.#6#6#6#6#.....C&H|__",
+        "d_|.....#6#6#.....6..|_d",
+        "__|......c.c......C..|__",
+        "__|||||...........C..|__",
+        "__|...D..c.c......C..|__",
+        "__|tCS|.#6#6#.v{{...f|__",
+        "_d||||||||||||||||||||__",
+        "______________d_________",
+        "______d_d_____________d_"
+      ],
+      "terrain": {
+        "#": "t_floor",
+        "&": "t_floor",
+        "+": "t_reinforced_door_glass_c",
+        "-": "t_wall_glass",
+        ".": "t_floor",
+        "6": "t_console",
+        "C": "t_floor",
+        "D": "t_door_c",
+        "H": "t_floor",
+        "P": "t_wall_glass",
+        "S": "t_floor",
+        "_": "t_grass",
+        "c": "t_floor",
+        "d": "t_dirt",
+        "f": "t_floor",
+        "p": "t_pavement",
+        "s": "t_sidewalk",
+        "t": "t_floor",
+        "v": "t_floor",
+        "{": "t_floor",
+        "|": "t_brick_wall"
+      }
+    },
+    "om_terrain": "lancenter",
+    "type": "mapgen",
+    "weight": 100
+  }
+]
```

Comment:
EOL missing.

Reply:
EOL?

Reply:
End of line mark, usually inserted by your text editor. What do you use?

Reply:
Notepad++

Reply:
Just put a extra row at the end of the JSON and it should fix the issue

Reply:
thanks, changes mades